### DISCUSSION TITLE
how i can trigger 2 commands by one click

### DISCUSSION THREAD

**Discussion by Adamkaram** [gdh1995/vimium-c#626] [Q&A] [answered]
iss there's any possiable way to trigger these command with one key
map xxx sortTabs sort="host,recency"
map xxx  joinTabs order="'host'"

**Comment by gdh1995** [comment, marked as answer]
Usually https://github.com/gdh1995/vimium-c/wiki/Auto-run-a-tree-of-commands will help.

But in this case, in fact `sortTabs` and `joinTabs` are a same command with different default parameters, and `sortTabs`  means `joinTabs sort="recency" windows="current"`, so `joinTabs order="host"` should just be enough.

**Reply by Adamkaram** [reply]
i believe that joinTabs order="host" didn't work well with me , it just merge no oder , i need to sort all tabs after oredeting

**Reply by gdh1995** [reply]
Thanks for this report. It seems a bug.

**Reply by gdh1995** [reply]
```map xxx joinTabs order="host" windows="all"``` will work - `windows="all"` is necessary to make tabs in a current window sorted.

I'll change this logic so that `order` or `sort` will indicate `windows="all"` (when there's no `windows` parameter).

**Reply by Adamkaram** [reply]
yup it's working now ,  and what do u mean about that 
"I'll change this logic so that order or sort will indicate windows="all" (when there's no windows parameter)."

**Reply by gdh1995** [reply]
In a next version, `map xxx joinTabs order="****"` will work just like `map xxx joinTabs order="***" windows="all"`

**Reply by Adamkaram** [reply]
better than before
